Makefile

```
@@ -440,3 +440,223 @@ docker-image-local: ## Build the eval-hub Docker image locally from Containerfil
+	@echo "Sending MCP initialize request..."
+	@response=$$(curl -sf -X POST http://localhost:$(MCP_TEST_PORT)/mcp \
+		-H "Content-Type: application/json" \
+		-d '{"jsonrpc":"2.0","id":1,"method":"initialize","params":{"protocolVersion":"2025-03-26","capabilities":{},"clientInfo":{"name":"test","version":"0.0.1"}}}' 2>&1) || \
+		{ echo "FAIL: MCP initialize request failed"; $(DOCKER) logs $(MCP_TEST_CONTAINER); $(DOCKER) rm -f $(MCP_TEST_CONTAINER); exit 1; }; \
+	echo "Response: $$response"; \
+	echo "$$response" | grep -q "serverInfo" || \
+		{ echo "FAIL: response missing serverInfo"; $(DOCKER) rm -f $(MCP_TEST_CONTAINER); exit 1; }
+	@$(DOCKER) rm -f $(MCP_TEST_CONTAINER)
+	@echo "PASS: container starts in HTTP mode and responds to MCP initialize"
+
+test-mcp-checksums: ## Generate SHA256 checksums and verify they match binaries
```

checkmake (0.3.2)

[warning] Target body for "test-mcp-checksums" exceeds allowed length of 5 lines (8).

(maxbodylength)